extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -764,12 +764,13 @@ export class AmpStoryInteractive extends AMP.BaseElement {
     );
 
     this.optionsData_ = data;
-    data.forEach((response, index) => {
+    this.orderData_();
+    data.forEach((response) => {
```

Comment:
Can you iterate here over `this.optionsData_`? The reasoning behind this is that currently, `response.index` could be out of bounds, but on `this.optionsData` it will filter out that so if the backend responds `{index: 10, selected: true}` we ignore it instead of erroring out

Reply:
Yeah, that definitely makes a lot more sense!